### ISSUE TITLE
Add documentation for wgl?

### ISSUE THREAD

**Issue by fryeb** [BSVino/docs.gl#57]
Just a suggestion, I personally would find better wgl documentation quite useful, as the existing wgl documentation is quite poor.

**Comment by tilkinsc** [comment]
I agree. I want to add this into my bindings library, something short and sweet like docs.gl/wgl/wglCreateContext
